src/Slack/MessageSender.php

```diff
@@ -170,9 +184,15 @@ public function sendAdminMessage(SecretSanta $secretSanta, string $code, string
 ```%s```
 - Paste the content on <%s|this page> then submit
 
-Remember, with great power comes great responsibility!
+Remember, with great power comes great responsibility!' . \PHP_EOL;
+
+        if ($scheduled) {
+            $message .= 'The messages will be sent at this time : ' . date('H:i - m/d/Y', $secretSanta->getOptions()['scheduled_at']) . \PHP_EOL;
```

Comment:
Maybe we can precise the UTC timezone in the summary to not confuse the user

Reply:
It was but I must've lost it in the squash. It's back now!